devpy/cmds/_build.py

```diff
@@ -29,18 +26,17 @@ def build_meson(build_dir, meson_args, jobs=None, clean=False, verbose=False):
 
     CFLAGS="-O0 -g" ./dev.py build
     """
-    build_dir = os.path.abspath(build_dir)
-    inst_dir = install_dir(build_dir)
+    build_dir = os.path.abspath("build")
```

Comment:
Quick Question: Does this mean we loose the flexibility to use different names for the build directory? I think for Meson it's OK as it defaults to `"build"` but other build systems in the future may not?

Reply:
I think the next refactoring I'll do will separate meson build & test from `build`-based build & test, with perhaps other options to come. The meson test process is fundamentally different from how you would approach it with installed wheels or editable installs. Eventually, they may come together again, when Meson's editable install is properly fleshed out, we'll see.